nibabel/viewers.py

```diff
@@ -118,8 +118,8 @@ def __init__(self, data, affine=None, axes=None, title=None):
             if self.n_volumes <= 1:
                 fig.delaxes(self._axes[3])
                 self._axes.pop(-1)
-            if self._title is not None:
-                fig.canvas.set_window_title(str(title))
+            if self._title is not None and fig.canvas.manager is not None:
```

Comment:
Is there a case where `fig.canvas.manager` can be `None`?

Reply:
I think if you instantiate the figure instance yourself (e.g., to embed in a Qt application of your own) instead of going through mpl it's probably None. I haven't checked but there is probably some condition like this that causes it to be None.

Reply:
Verified that indeed `fig.canvas.manager is None` if you embed on your own. But then I noticed that this code is in a block where we directly create `fig` using `pyplot`, so indeed we should be good to remove the conditional (I hope/presume)!